plone/dexterity/schema.py

```diff
@@ -49,10 +49,13 @@ def volatile(func):
     def decorator(self, portal_type):
         """lookup fti from portal_type and cache
         """
-        if IDexterityFTI.providedBy(portal_type):
-            fti = portal_type
+        if portal_type is not None:
```

Comment:
Could this be ``if portal_type:`` or there is a specific reason for checking against ``None``? An empty string should go inside this conditional as well?